tests/libvmi/factory.go

```diff
@@ -51,11 +53,29 @@ func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return New(NamespaceTestDefault, RandName(DefaultVmiName), opts...)
 }
 
+// NewCirros instantiates a new Cirros based VMI configuration,
+// building its extra properties based on the specified With* options.
+func NewCirros(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	// Without a cloud-init section, the VM gets stuck on the boot, waiting to
+	// receive an IP from a DHCP server. When cloud-init is used, it gets an IP
+	// immediatelly.
+	echo := `#!/bin/bash
+	echo `
```

Comment:
maybe better some `echo Hello world` ?

Reply:
How is it better?

Reply:
just maybe it would look less weird than empty echo and more self explanatory (also if someone would copy it, the reason would be embedded)
up to you